.github/config.yml

```diff
@@ -0,0 +1,40 @@
+# Configuration for request-info - https://github.com/behaviorbot/request-info
+
+# *OPTIONAL* Comment to reply with
+# Can be either a string :
+# requestInfoReplyComment: >
+#   We would appreciate it if you could provide us with more info about this issue/pr!
+
+# Or an array:
+requestInfoReplyComment:
```

Comment:
What is that for? Do you have a picture of how it will look in the UI?

Reply:
it requests info for more data if there is less info. I recommend some issue templates, I will make it. It can not be made so immediately.

Reply:
It needs an app btw